docs/_data/sidebars/left-navigation-sidebar.yml

```diff
@@ -240,6 +240,10 @@ entries:
       url: /resources/ui-kit.html
       output: web
 
+    - title: SAP Fiori Fundamentals Libraries
```

Comment:
Should we make this "SAP Fiori Fundamentals Implementation Libraries" so it's not confusing since Fiori Fundamentals _is_ a library in itself?